### ISSUE TITLE
Version number was not updated in release 0.3.21

### ISSUE THREAD

**Issue by SamsonovAnton** [Thomas-Tsai/partclone#212]
Still 0.3.20 in
https://github.com/Thomas-Tsai/partclone/blob/608a9bfb1c1da99cb3b08c0e92056ef20a8e6916/configure.ac#L1

Could you please fix this and re-release while it's still not too late?

**Comment by robert-scheck** [comment]
Please do not re-release an existing release (especially not days after the release itself); partclone 0.3.21 release tarball already has been used in Linux distributions. Instead, please do it correctly for the next release.

**Comment by Thomas-Tsai** [comment]
Yes, Thanks for your reminder!
We notice that and are ready to release 0.3.22 combined with the recent changes!

**Comment by robert-scheck** [comment]
No, this is unfortunately not solved: [configure.ac:1](https://github.com/Thomas-Tsai/partclone/blob/master/configure.ac#L1) still says "0.3.20" instead of "0.3.22".

**Comment by SamsonovAnton** [comment]
> [configure.ac:1](https://github.com/Thomas-Tsai/partclone/blob/master/configure.ac#L1) still says "0.3.20" instead of "0.3.22".

That's puzzling, since [released tarball](https://github.com/Thomas-Tsai/partclone/releases/tag/0.3.22) indeed contained "0.3.22" almost 31 hours earlier — both in source code and the binaries compiled from it.
